### DISCUSSION TITLE
how to fill complex PolygonLayer with overlapping lat/lng bounds?

### DISCUSSION THREAD

**Discussion by rammoozz** [visgl/deck.gl#5941] [Q&A]
While attempting to draw an hour glass or any shape with more than 3 points that cross over bounds, the PolygonLayer doesn't seem to fill correctly, is there a property to aid with this? 

https://codesandbox.io/s/deckgl-forked-96pb4
![image](https://user-images.githubusercontent.com/5125481/124150673-c2ce9980-da5f-11eb-9514-15f7d8781681.png)

Expectation (Google Maps Polygon):
https://codesandbox.io/s/bold-mayer-qwx69?file=/src/index.ts
![image](https://user-images.githubusercontent.com/5125481/124803414-13923680-df27-11eb-87ca-785dea8a524f.png)

**Comment by felixpalmer** [comment]
That's not a valid polygon, if you want to draw that shape draw it as two separate triangles

**Reply by VictorVelarde** [reply]
...or maybe use an IconLayer? https://deck.gl/docs/api-reference/layers/icon-layer

**Reply by rammoozz** [reply]
It is a _valid_ polygon, but the coordinates are not in the correct order (hence the overlap, its an hourglass), google maps renders it correctly too. [Google maps sandbox link](https://codesandbox.io/s/bold-mayer-qwx69?file=/src/index.ts)

**Reply by Pessimistress** [reply]
See discussion in https://github.com/visgl/deck.gl/issues/4981#issuecomment-701114382
